src/reference/config/guards/jwt.md

```diff
@@ -117,66 +91,52 @@ Audience claim.
 
 #### options.challenge
 
-> `integer`
+> `number`
 
 Challenge period (seconds).
```

Comment:
`challenge` is a whole number of seconds, so floating point numbers are not valid.

Reply:
so this should be updated in the scheme then?
```
"challenge": {
                                        "title": "Challenge",
                                        "anyOf": [
                                            {
                                                "type": "number"
                                            },
                                            {
                                                "$ref": "#/$defs/expression"
                                            }
                                        ]
                                    }
```

Reply:
Yeah, I'd say so.

Reply:
added a PR and updated the docs